cubic-server/thread_pool/PriorityThreadPool.hpp

```diff
@@ -114,7 +114,7 @@ class PriorityThreadPool {
         {
             const std::lock_guard<std::mutex> _guard(_toolBox.queueProtection);
             jobId = ++_toolBox.totalJobsPushed;
-            _toolBox.jobQueue.emplace_back(jobId, priority, std::move(realJobList));
+            _toolBox.jobQueue.emplace_back(PriorityThreadPoolUtility::Job {jobId, priority, std::move(realJobList)});
```

Comment:
Why create an object here for emplace? It was working fine without, and it would construct the object inplace in the jobqueue, which is faster.

Reply:
I think it was a clang error ? But it was so long ago that this is a total guess🤡

Reply:
I'll check today

Reply:
I was right, this generate an error with clang, but only because the struct was declared in a weird way...